Lesson33/props-drilling-app/Gokhan/src/components/Account/index.jsx

```diff
@@ -0,0 +1,16 @@
+import { UserDetailsForm } from '../UserDetailsForm';
+import { useState } from 'react';
+
+export const Account = () => {
+  const [username, setUsername] = useState('Placeholder User Name');
```

Comment:
Remove this line, use state is kept in App component as you need it both in Account, UserDetailsForm and Navbar.